jenkinsapi_tests/systests/test_quiet_down.py

```diff
@@ -0,0 +1,11 @@
+"""
+System tests for setting jenkins in quietDown mode
+"""
+
+
+def test_quiet_down(jenkins):
```

Comment:
There are no test assertions. What is the point of this test?

Reply:
The purpose is to make sure the added functionality gets executed during testing. Unfortunately, there is currently no support (as far as I have been able to see) in the code for requesting <server_url>/api/xml 
This would make it possible to check the value of:  <quietingDown>false</quietingDown>

Reply:
I'can see Travis is failing everything, but I'm not able to figure out, what I can do to fix this. Is it caused by my changes?